scout/server/links.py

```diff
@@ -714,7 +714,7 @@ def ucsc_link(variant_obj, build=None):
     )
     if build == 38:
         url_template = (
-            "http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg20&"
+            "http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&"
```

Comment:
20? weird!

Reply:
I know! Very much wonder if it was ever like that on their side or if it is our typo…